wpe/src/main/cpp/browser/messagepump.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include <glib.h>
+#include <memory>
+#include <vector>
+
+struct ALooper;
+
+class MessagePump
+{
+public:
+    MessagePump();
```

Comment:
If you override a default constructor, you have to explicitly override all (copy and move constructors/operators + destructor).

Maybe you can keep it like this for now as this kind of thing will be detected by clang-tidy and we may have other places to correct too. So we may fix it later when clang-tidy is configured.

Reply:
done